NOTICE

```diff
@@ -4,86 +4,47 @@ Copyright 2024 The Apache Software Foundation
-      spark/datasourcs-33/src/main/scala/org/apache/graphar/datasources/orc/ORCWriteBuilder.scala
-      spark/datasourcs-33/src/main/scala/org/apache/graphar/datasources/parquet/ParquetWriteBuilder.scala
-    are modified from spark.
-
-* arrow (https://github.com/apache/arrow)
-* fastFFI (https://github.org/apache/fastFFI)
-    Files:
-      java/src/main/java/org/apache/graphar/stdcxx/StdString.java
-    are modified from fastFFI.
-
-* GraphScope (https://github.org/apache/GraphScope)
-    Files:
-      java/src/main/java/org/apache/graphar/stdcxx/StdVector.java
-      java/src/main/java/org/apache/graphar/stdcxx/StdMap.java
-      java/src/main/java/org/apache/graphar/stdcxx/StdSharedPtr.java
-      java/src/main/java/org/apache/graphar/stdcxx/StdUnorderedMap.java
-      java/build.xml
-    are modified from GraphScope.
-
-
-================================================================
-MIT licenses
-================================================================
-The following components are provided under the MIT License. See project link for details.
-The text of each license is also included in licenses/LICENSE-[project].txt
-
-* result (https://github.com/bitwizeshift/result)
-    Files:
-      cpp/thirdparty/result.hpp is modified from result.
-
-* mini-yaml (https://github.com/jimmiebergmann/mini-yaml)
-    Files:
-      cpp/thirdparty/mini-yam/Yaml.hpp
-      cpp/thirdparty/mini-yaml/Yaml.cpp
-
-* simple-uri-parser (https://github.com/jholloc/simple-uri-parser)
-    Files:
-      cpp/thirdparty/simple-uri-parser/uri_parser.h
-
-
-================================================================
-BSD 3-Clause licenses
-================================================================
-The following components are provided under the BSD 3-Clause License. See project link for details.
-The text of each license is also included in licenses/LICENSE-[project].txt
-
-* cpplint (https://github.com/cpplint/cpplint)
-    Files:
-      cpp/misc/cpplint.py
+This product includes software from Apache Arrow project (Apache 2.0)
+  * Apache Arrow
+  * Copyright 2016-2019 The Apache Software Foundation
+
+--------------------------------------------------------------------------------
+
+This product includes code from Apache Spark, which includes the following in
+its NOTICE file:
+
+  Apache Spark
+  Copyright 2014 and onwards The Apache Software Foundation.
+
+--------------------------------------------------------------------------------
+
+This product includes code from https://github.com/alibaba/GraphScope (Apache 2.0),
+which includes the following in its NOTICE file:
+
+  GraphScope
+  Copyright 2020 Alibaba Group Holding Limited. 
+
+--------------------------------------------------------------------------------
+
+This product includes code from https://github.com/alibaba/fastFFI (Apache 2.0)
+  * Copyright 1999-2021 Alibaba Group Holding Ltd.
+
```

Comment:
If the third-party code does not have a corresponding NOTICE file, it is not necessary to add one.

Reply:
Thanks for pointing out, already removed the third-party dependency without NOTICE file form NOTICE